### PR TITLE
Fix Ansible Lint Job

### PR DESCRIPTION
**Pull request by Mab879**
#### Description:
* Fix no pipefail
* Reident the Ansible lint file
* Move some rules to skip so the results are actually readable

#### Rationale:
* Make the job pass
* Make the job useful

### CODE REVIEW

File: shared/macros/10-ansible.jinja [hunk 1]
```diff
@@ -1544,7 +1549,9 @@ Part of the grub2_bootloader_argument_absent template.
 {{% macro ansible_ensure_authselect_custom_profile() -%}}
 - name: '{{{ rule_title }}} - Get authselect current profile'
   ansible.builtin.shell:
-    cmd: authselect current -r | awk '{ print $1 }'
+    cmd:
+        set -o pipefail
+        authselect current -r | awk '{ print $1 }'
```
Comment (jan-cerny) [inline shared/macros/10-ansible.jinja:1554]: I think you're creating an invalid YAML, there needs to be `|`  or `|-` after `cmd: ` to create a multiline string.

**Review by jan-cerny — COMMENTED** [review]
The testing farm fail looks legit:

```

TASK [Lock Accounts After Failed Password Attempts - Create an authselect custom profile based on the current profile] ***
fatal: [localhost]: FAILED! => {"changed": true, "cmd": ["authselect", "create-profile", "hardening", "-b"], "delta": "0:00:00.002448", "end": "2023-11-09 22:18:32.143774", "msg": "non-zero return code", "rc": 1, "start": "2023-11-09 22:18:32.141326", "stderr": "Invalid option -b: missing argument\n\nUsage: authselect create-profile NAME [OPTIONS...]\n\nCommand options:\n  -v, --vendor               Create new profile as a vendor profile instead of\n                             a custom profile\n  -b, --base-on=STRING       ID of a profile that should be used as a base for\n                             the new profile\n      --base-on-default      Base new profile on a default profile even if\n                             vendor profile with the same name exists\n      --symlink-meta         Symlink meta files from the base profile instead\n                             of copying them\n      --symlink-nsswitch     Symlink nsswitch files from the base profile\n                             instead of copying them\n      --symlink-pam          Symlink pam files from the base profile instead\n                             of copying them\n      --symlink-dconf        Symlink dconf files from the base profile instead\n                             of copying them\n  -s, --symlink              Symlink specific file (can be set multiple times)\n\nCommon options:\n      --debug                Print error messages\n      --trace                Print trace messages\n      --warn                 Print warning messages\n\nHelp options:\n  -?, --help                 Show this help message\n      --usage                Display brief usage message\n[error] Unable to parse command arguments", "stderr_lines": ["Invalid option -b: missing argument", "", "Usage: authselect create-profile NAME [OPTIONS...]", "", "Command options:", "  -v, --vendor               Create new profile as a vendor profile instead of", "                             a custom profile", "  -b, --base-on=STRING       ID of a profile that should be used as a base for", "                             the new profile", "      --base-on-default      Base new profile on a default profile even if", "                             vendor profile with the same name exists", "      --symlink-meta         Symlink meta files from the base profile instead", "                             of copying them", "      --symlink-nsswitch     Symlink nsswitch files from the base profile", "                             instead of copying them", "      --symlink-pam          Symlink pam files from the base profile instead", "                             of copying them", "      --symlink-dconf        Symlink dconf files from the base profile instead", "                             of copying them", "  -s, --symlink              Symlink specific file (can be set multiple times)", "", "Common options:", "      --debug                Print error messages", "      --trace                Print trace messages", "      --warn                 Print warning messages", "", "Help options:", "  -?, --help                 Show this help message", "      --usage                Display brief usage message", "[error] Unable to parse command arguments"], "stdout": "", "stdout_lines": []}
```

**Comment by Mab879** [comment]
Looks like the changes might be exposing some bugs (which is an intended side-effect) I will take look.

**Comment by jan-cerny** [comment]
There is one more fail on CS 9:

```

TASK [Enable SSH Server firewalld Firewall Exception - Ensure NetworkManager connections are assigned to a firewalld zone] ***
failed: [localhost] (item=['5fb06bd0-0bb0-7ffb-45f1-d6edd65f3e03', {'changed': False, 'stdout': '--', 'stderr': '', 'rc': 0, 'cmd': "set -o pipefail\nset -o pipefail && nmcli -f connection.zone connection show 5fb06bd0-0bb0-7ffb-45f1-d6edd65f3e03 | awk '{ print $2}'", 'start': '2023-11-16 16:46:15.387884', 'end': '2023-11-16 16:46:15.443604', 'delta': '0:00:00.055720', 'msg': '', 'invocation': {'module_args': {'_raw_params': "set -o pipefail\nset -o pipefail && nmcli -f connection.zone connection show 5fb06bd0-0bb0-7ffb-45f1-d6edd65f3e03 | awk '{ print $2}'", '_uses_shell': True, 'stdin_add_newline': True, 'strip_empty_ends': True, 'argv': None, 'chdir': None, 'executable': None, 'creates': None, 'removes': None, 'stdin': None}}, 'stdout_lines': ['--'], 'stderr_lines': [], 'failed': False, 'item': '5fb06bd0-0bb0-7ffb-45f1-d6edd65f3e03', 'ansible_loop_var': 'item'}]) => {"ansible_loop_var": "item", "changed": false, "cmd": "set -o pipefail nmcli connection modify 5fb06bd0-0bb0-7ffb-45f1-d6edd65f3e03 connection.zone public", "item": ["5fb06bd0-0bb0-7ffb-45f1-d6edd65f3e03", {"ansible_loop_var": "item", "changed": false, "cmd": "set -o pipefail\nset -o pipefail && nmcli -f connection.zone connection show 5fb06bd0-0bb0-7ffb-45f1-d6edd65f3e03 | awk '{ print $2}'", "delta": "0:00:00.055720", "end": "2023-11-16 16:46:15.443604", "failed": false, "invocation": {"module_args": {"_raw_params": "set -o pipefail\nset -o pipefail && nmcli -f connection.zone connection show 5fb06bd0-0bb0-7ffb-45f1-d6edd65f3e03 | awk '{ print $2}'", "_uses_shell": true, "argv": null, "chdir": null, "creates": null, "executable": null, "removes": null, "stdin": null, "stdin_add_newline": true, "strip_empty_ends": true}}, "item": "5fb06bd0-0bb0-7ffb-45f1-d6edd65f3e03", "msg": "", "rc": 0, "start": "2023-11-16 16:46:15.387884", "stderr": "", "stderr_lines": [], "stdout": "--", "stdout_lines": ["--"]}], "msg": "[Errno 2] No such file or directory: b'set'", "rc": 2, "stderr": "", "stderr_lines": [], "stdout": "", "stdout_lines": []}
failed: [localhost] (item=['ed254861-812b-4c79-9a56-1b7a21545d91', {'changed': False, 'stdout': '--', 'stderr': '', 'rc': 0, 'cmd': "set -o pipefail\nset -o pipefail && nmcli -f connection.zone connection show ed254861-812b-4c79-9a56-1b7a21545d91 | awk '{ print $2}'", 'start': '2023-11-16 16:46:15.688283', 'end': '2023-11-16 16:46:15.744028', 'delta': '0:00:00.055745', 'msg': '', 'invocation': {'module_args': {'_raw_params': "set -o pipefail\nset -o pipefail && nmcli -f connection.zone connection show ed254861-812b-4c79-9a56-1b7a21545d91 | awk '{ print $2}'", '_uses_shell': True, 'stdin_add_newline': True, 'strip_empty_ends': True, 'argv': None, 'chdir': None, 'executable': None, 'creates': None, 'removes': None, 'stdin': None}}, 'stdout_lines': ['--'], 'stderr_lines': [], 'failed': False, 'item': 'ed254861-812b-4c79-9a56-1b7a21545d91', 'ansible_loop_var': 'item'}]) => {"ansible_loop_var": "item", "changed": false, "cmd": "set -o pipefail nmcli connection modify ed254861-812b-4c79-9a56-1b7a21545d91 connection.zone public", "item": ["ed254861-812b-4c79-9a56-1b7a21545d91", {"ansible_loop_var": "item", "changed": false, "cmd": "set -o pipefail\nset -o pipefail && nmcli -f connection.zone connection show ed254861-812b-4c79-9a56-1b7a21545d91 | awk '{ print $2}'", "delta": "0:00:00.055745", "end": "2023-11-16 16:46:15.744028", "failed": false, "invocation": {"module_args": {"_raw_params": "set -o pipefail\nset -o pipefail && nmcli -f connection.zone connection show ed254861-812b-4c79-9a56-1b7a21545d91 | awk '{ print $2}'", "_uses_shell": true, "argv": null, "chdir": null, "creates": null, "executable": null, "removes": null, "stdin": null, "stdin_add_newline": true, "strip_empty_ends": true}}, "item": "ed254861-812b-4c79-9a56-1b7a21545d91", "msg": "", "rc": 0, "start": "2023-11-16 16:46:15.688283", "stderr": "", "stderr_lines": [], "stdout": "--", "stdout_lines": ["--"]}], "msg": "[Errno 2] No such file or directory: b'set'", "rc": 2, "stderr": "", "stderr_lines": [], "stdout": "", "stdout_lines": []}
```

**Comment by Mab879** [comment]
This will need more work, closing for now.

**Pull request closed without merging**